pages/common/lpass.md

```diff
@@ -0,0 +1,23 @@
+# lpass
+
+> Command line interface for LastPass Password Manager.
+
+- Login to your LastPass account, by entering your Master Password when promted:
+
+`lpass login {{username}}`
+
+- Show login status:
+
+`lpass status`
+
+- List all sites grouped by category:
+
+`lpass ls`
+
+- Show password for a specified entry (usually the website url):
+
+`lpass show {{identifier}} --password`
+
+- Generate a new password and add to LastPass:
```

Comment:
This should go above the previous example. And let's use concrete examples for url and identifier. And use the same identifier for the `show` command.

Also typo at password_lenght